### DISCUSSION TITLE
Package WebView2 with Air app?

### DISCUSSION THREAD

**Discussion by colm55** [airsdk/Adobe-Runtime-Support#2232] [Q&A]
Hi,

Is it possible to package a specific/fixed instance of WebView2 with an Air app (run-time embedded option) so that the Air app has access to WebView2 even on computers without WebView2 installed?

Thanks,
Colm.

**Comment by 2jfw** [comment]
Without using a third party installer you will not be able to package the WebView2 within an AIR app.
Using a third party installer you will need to run/install the standalone version of the WebView2 **Evergreen Standalone Installer** (https://developer.microsoft.com/en-us/microsoft-edge/webview2/).
I think you can do that silently - just search for something like "silent mode for installing prerequisites".

**Reply by colm25** [reply]
Thanks Jan.
I was afraid of that! My "users" have a low level of privilege, and certainly wouldn't have privs for this.
Oh well ... Back onto IT Services I guess!
